### ISSUE TITLE
`minpac` doesn't seem to work well

### ISSUE THREAD

**Issue by m-ou-se** [m-ou-se/config#14]
Somehow, `minpac` seems to miss some packages while installing or updating sometimes. On a new computer, I often have to run the install script multiple times until I get all plugins.

**Comment by m-ou-se** [comment]
Using `call minpac#update('', {'do': 'quit'})` instead of using `+qall` on the command line doesn't change anything. It still seems to randomly miss some packages.

**Comment by de-vri-es** [comment]
yeah I have the same thing. We could just use submodules for vim packs, but I don't really care for tracking submodule hashes. Going to get annoying to update those.

Alternatively, but also not necessarily a big fan, we could just clone/update vim plugins in the install script.

We could also switch back to `vim-plug`.

**Comment by de-vri-es** [comment]
Made the install script update minpac and use `minpac#update('', {'do': 'quit'})`. Seems to work well now. Maybe it was also a bug in old minpac versions.

However, the second time vim is run the ShowMarks plugin is doing something annoying that requires you to press enter. That kinda messes up installing coc extensions :(
